condDatasetSubmitter.py

```diff
@@ -704,6 +710,14 @@ def printInfo(options):
 #-------------------------------------------------------------------------------
 
 if __name__ == "__main__":
+  #Raise an error if couchID files exist
+  p = subprocess.Popen("ls", stdout=subprocess.PIPE, shell=True)
+  out = p.stdout.read().strip()
+  newlist = out.split('\n')
+  substring = ".couchID"
+  for object in newlist:
+    if substring in object:
+      raise ValueError("couchID file exists, please remove it");
```

Comment:
Hi @franzoni then wouldn't it make more sense for the program to give you the list of files that gives problems? Also, the python can stop execution and ask the user how to proceed